src/app/(with-header-sidebar)/dashboard/[id]/components/Column.module.css

```
@@ -0,0 +1,97 @@
+.column {
+  display: flex;
+  flex-direction: column;
+  width: 354px;
+  height: 1010px;
+  padding: 20px;
+  border-right: 1px solid var(--gray-200);
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.status {
+  display: flex;
+  align-items: center;
+}
+
+.dot {
+  width: 8px;
+  height: 8px;
+  border-radius: 999px;
```

```suggestion
  border-radius: 50%;
```

원을 만드시는 것이면 50%가 더 명시적입니다.